src/main/java/exception/DukeException.java

```diff
@@ -0,0 +1,37 @@
+package exception;
+
+/**
+ * Represents all exception unique to Duke.
+ */
+public class DukeException extends Exception {
+    protected String type;
+
+    /**
+     * Initializes DukeException with error type.
+     * @param input the error type
```

Comment:
Minor Javadoc style violation as there should be an [empty line between description and parameter section](https://nus-cs2103-ay1920s1.github.io/website/coding-standards/java/intermediate.html#comments). This style is adhered to in other Javadoc comments so I believe this style violation was unintended.